docs/test-fairy/security/data-encryption.md

```diff
@@ -8,7 +8,62 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-/Security/End_to_End_Data_Encryption.html
+TestFairy provides a mechanism to encrypt the logs and screenshots recorded from the mobile device. This way, the data kept on the cloud is encrypted, and nobody can read it other than the people in your team who have the private key.
+To use this capability, you will need to create a public key and a private key. The public key initializes the TestFairy SDK in your app. The private key, which should not be shared with anybody, will be used by you when you log in to your TestFairy dashboard.
 
-https://github.com/testfairy/docs/blob/master/docs/610_Security/200_End_to_End_Data_Encryption.md
+This encryption is done with a randomly generated 256-bit AES key. This AES key is random and is only used in a single session recording. The AES key is then protected with an RSA key, where the public key is provided when constructing the SDK.
 
+## Generating Public/Private Key Pair
+
+Generating a pair is done using `openssl` tool:
+
+```bash
+openssl genrsa -out testfairy-private-key.pem 2048
+openssl rsa -in testfairy-private-key.pem -outform DER -pubout | base64 - > testfairy-public-key.txt
+```
+
+It will create two files called `testfairy-private-key.pem` and `testfairy-public-key.txt` containing your private and public keys.
+
+The file `testfairy-private-key.pem` is sensitive and should not be shared with anyone outside your team.
+
+The content of `testfairy-public-key.txt` will be used to initialize the SDK. Paste this value into the first parameter of `TestFairy.setPublicKey` method.
+
+## Using Android
+
+Enable end-to-end encryption for your Android apps by calling `setPublicKey` before calling the `begin` method.
+
+```java
+TestFairy.setPublicKey("<PUBLIC KEY>");
+TestFairy.begin("<APP TOKEN>");
+```
+
+## Using iOS
+
+Enable end-to-end encryption for your iOS apps by calling `setPublicKey` before calling the `begin` method.
+
+```js
+[TestFairy setPublicKey:@"<PUBLIC KEY>"];
+[TestFairy begin:@"<APP TOKEN>"];
+```
+
+## Viewing encrypted sessions
+
+Since the data is encrypted using RSA, viewing a session requires the private key. Simply visiting a recorded session will prompt a dialog for entry of the RSA Private Key. Just paste the private key text and click "OK". Your private keys are never sent to the server and are only kept within the browser session.
```

Comment:
```suggestion
Since the data is encrypted using RSA, viewing a session requires the private key. Visiting a recorded session will prompt a dialog for entry of the RSA Private Key. Just paste the private key text and click "OK". Your private keys are never sent to the server and are only kept within the browser session.
```